tests/test_wave_06.py

```diff
@@ -1,63 +1,47 @@
-from app.models.goal import Goal
+from app.models import Goal
 import pytest
 
 
-@pytest.mark.skip(reason="No way to test this feature yet")
 def test_post_task_ids_to_goal(client, one_goal, three_tasks):
     # Act
-    response = client.post("/goals/1/tasks", json={
-        "task_ids": [1, 2, 3]
-    })
+    response = client.post("/goals/1/tasks", json={"task_ids": [1, 2, 3]})
     response_body = response.get_json()
 
     # Assert
     assert response.status_code == 200
     assert "id" in response_body
     assert "task_ids" in response_body
-    assert response_body == {
-        "id": 1,
-        "task_ids": [1, 2, 3]
-    }
+    assert response_body == {"id": 1, "task_ids": [1, 2, 3]}
 
     # Check that Goal was updated in the db
     assert len(Goal.query.get(1).tasks) == 3
 
 
-@pytest.mark.skip(reason="No way to test this feature yet")
-def test_post_task_ids_to_goal_already_with_goals(client, one_task_belongs_to_one_goal, three_tasks):
+def test_post_task_ids_to_goal_already_with_goals(
+    client, one_task_belongs_to_one_goal, three_tasks
+):
     # Act
-    response = client.post("/goals/1/tasks", json={
-        "task_ids": [1, 4]
-    })
+    response = client.post("/goals/1/tasks", json={"task_ids": [1, 4]})
     response_body = response.get_json()
 
     # Assert
     assert response.status_code == 200
     assert "id" in response_body
     assert "task_ids" in response_body
-    assert response_body == {
-        "id": 1,
-        "task_ids": [1, 4]
-    }
+    assert response_body == {"id": 1, "task_ids": [1, 4]}
```

Comment:
Heck yeah, I love that you improved the style to match yours. Sometimes your hand will be forced to remain consistent with others' code, but I think it's fine in this case.

Reply:
i blame black formatter